### ISSUE TITLE
`hookpoint` v2.3.1 broke httpyac

### ISSUE THREAD

**Issue by yanghua-ola** [AnWeber/httpyac#470]
To reproduce:
1. install httpyac 6.4.4 freshly (it will download latest version of `hookpoint`, v2.3.1)
2. run `httpyac send` command with any http file, and expect this error
```
TypeError: Cannot read properties of undefined (reading 'progress')
    at b (/Users/yanghua/repos/agilekit-new/node_modules/httpyac/dist/index.js:2:18182)
    at Mr.beforeLoop (/Users/yanghua/repos/agilekit-new/node_modules/httpyac/dist/index.js:2:54617)
    at p.intercept (/Users/yanghua/repos/agilekit-new/node_modules/hookpoint/dist/index.js:1:3710)
    at async p.trigger (/Users/yanghua/repos/agilekit-new/node_modules/hookpoint/dist/index.js:1:3022)
    at async Se.execute (/Users/yanghua/repos/agilekit-new/node_modules/httpyac/dist/index.js:2:49483)
    at async Yd (/Users/yanghua/repos/agilekit-new/node_modules/httpyac/dist/index.js:119:29721)
    at async Bm (/Users/yanghua/repos/agilekit-new/node_modules/httpyac/dist/index.js:119:29966)
    at async vo (/Users/yanghua/repos/agilekit-new/node_modules/httpyac/dist/index.js:119:29473)
    at async /Users/yanghua/repos/agilekit-new/node_modules/httpyac/dist/index.js:119:38064
    at async o (/Users/yanghua/repos/agilekit-new/node_modules/httpyac/dist/index.js:2:38791)
```

**Comment by AnWeber** [comment]
@yanghua-ola I removed a parameter `arg` in hookpoint, but I should have fixed the spots. I will check it.

**Comment by AnWeber** [comment]
I have released a version 2.3.2 which should fix the error. Then I released another correct 3.0.0 with the breaking change
